app/services/users.js

```diff
@@ -26,3 +28,94 @@ exports.existUserMail = async mail => {
     return new Error(`signUpUser Error => ${error.message}`);
   }
 };
+
+exports.getUserByEmailAndPassword = async (mail, newPassword) => {
+  try {
+    let hasCorrectParameters = false;
+    const userData = await userModel.findOne({ where: { mail } });
+
+    if (objIsNotEmpty(userData) && userData.password) {
+      hasCorrectParameters = await checkPassword(newPassword, userData.password);
+    }
+
+    return hasCorrectParameters ? userData : {};
+  } catch (error) {
+    logger.error(`getUserByEmailAndPassword Error => ${error}`);
+    return new Error(`getUserByEmailAndPassword Error => ${error.message}`);
```

Comment:
I think that you could return a specific error using the wolox standar for example 

 `throw databaseError(`An error was occurred because the database is fail. error trace ${error.message}`)`

What do you think ?